.github/scripts/rust/Makefile.toml

```diff
@@ -158,3 +178,7 @@ install_crate = { rustup_component_name = "rustfmt" }
 private = true
 condition = { channels = ["nightly"] }
 install_crate = { rustup_component_name = "miri" }
+
+[tasks.install-nextest]
+private = true
+install_crate = { crate_name = "cargo-nextest", binary = "cargo", test_arg = ["nextest", "--version"] }
```

Comment:
`nextest` wouldn't be installed with `quick-install` when specifying it here, right? Do we want that, or is it fine here to use regular cargo install

Reply:
Yes, that's right, and intentionally. The CI makes sure, that `nextest` is installed beforehand. But if you run it locally, it will properly install it from source, as `quickinstall` is not necessarily available.